devicetypes/smartthings/fibaro-rgbw-controller.src/fibaro-rgbw-controller.groovy

```diff
@@ -26,562 +26,230 @@
 		capability "Switch Level"
 		capability "Actuator"
 		capability "Switch"
-		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
 		capability "Configuration"
 		capability "Color Control"
-        capability "Power Meter"
-
-        command "getDeviceData"
-        command "softwhite"
-        command "daylight"
-        command "warmwhite"
-        command "red"
-        command "green"
-        command "blue"
-        command "cyan"
-        command "magenta"
-        command "orange"
-        command "purple"
-        command "yellow"
-        command "white"
-        command "fireplace"
-        command "storm"
-        command "deepfade"
-        command "litefade"
-        command "police"
-        command "setAdjustedColor"
-        command "setWhiteLevel"
-        command "test"
-
-        attribute "whiteLevel", "string"
-
-		fingerprint deviceId: "0x1101", inClusters: "0x27,0x72,0x86,0x26,0x60,0x70,0x32,0x31,0x85,0x33"
+		capability "Color Temperature"
+		capability "Power Meter"
+		capability "Health Check"
```

Comment:
I forget, can we add the `Health Check` capability without also setting `checkInterval` at the same time?